docs/overview_API.ipynb

```diff
@@ -171,12 +170,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Next, read in your set of annual composites. To do this use the `read_and_stack_tifs` method provided in the package's `io.raster` module. The methods requires that you either provide a path to a directory containing all of your annual composites, or you provide a list of paths to each individual composite. Each composite:\n",
+    "Next, read in your set of annual composites. To do this use the `read_and_stack_tifs` method. The method requires that you either provide a path to a directory containing all of your annual composites, or you provide a list of paths to each individual composite. Each composite:\n",
     "\n",
     "1. _must_ be named after the year of which they are a composite, e.g for a composite of 2014 the file must be named `2014.tif`\n",
     "2. _must_ have the bands named by their common names/acronyms\n",
     "\n",
-    "Each composite can contain multiple bands but each band must be present in all composites."
+    "Each composite can contain multiple bands but each band must be present in all composites. You must also provide the Platform(s) from which the composite is derived.\n",
```

Comment:
Maybe more appropriate to call it Sensor? Although Platform is probably acceptable as well.